.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,90 @@
+name: Pull request
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+    - name: Build Apps DNS image
+      uses: docker/build-push-action@v2
+      with:
+        file: image/Dockerfile
+        tags: kubecf-apps-dns:latest
+        pull: true
+        push: false
+    - name: Build Acceptance tests image
+      uses: docker/build-push-action@v2
+      with:
+        file: tests/acceptance/image/Dockerfile
+        context: tests/acceptance/
+        tags: acceptance-tests:latest
+        pull: true
+        push: false
+    - uses: azure/setup-kubectl@v1
+    - uses: azure/setup-helm@v1
+    - name: Setup k3s
+      uses: SUSE/k3s-actions/create@master
```

Comment:
We really should be having a `v1` branch for that.  Also the other actions.  Otherwise it's a recipe for breaking this.

Reply:
Not a branch, but a tag. I don't think we are there just yet.

Reply:
`v1` should be a branch, not a tag, because it's not a fixed commit (it will change over time).  Though we might want to use `v0` instead if we're not ready to commit to an API yet.

Reply:
Fixed with 4fcf959.

As discussed outside of this PR, we want to consume tags, not branches, and let dependabot deal with the dependency bumps.